core/runtime/TRTEngine.cpp

```diff
@@ -10,6 +11,8 @@ namespace trtorch {
 namespace core {
 namespace runtime {
 
+const std::string empty_string = std::string();
```

Comment:
Is this necessary, can we pass in an anonymous empty string? If we do can we put this in an anonymous namespace?

Reply:
Why create an empty string again and again in a function call if we can reuse it?

Reply:
idk, seems odd honestly. like i feel like `new (this) TRTEngine(_name, serialized_engine, "")` or `new (this) TRTEngine(_name, serialized_engine, std::string())` is fine and the compiler will handle optimization

Reply:
Fixed it